static/docs/philosophy/video-resources.md

```diff
@@ -0,0 +1,25 @@
+## Video Resources 
+
+### Talks by DVC team
```

Comment:
subsections should be ## so that we show them in the right nav bar

Reply:
if it's the only subsection of the Video Resources, let's merge it withe the title